src/introduction/README.md

```diff
@@ -16,40 +18,40 @@ and software stacks.
-facilitate building skills such as compiling, executing,
-and debugging RISC-V and CHERI-RISC-V programs, as well as to build basic
-understanding of CHERI C/C++ properties. We highlight some key edge
-cases in CHERI, including the effects of bounds imprecision, subobject
-bounds, weaker temporal safety, and C type confusion.
+and ask you to build and perform minor modifications to simple C/C++
+programs. These exercises facilitate building skills such as
+compiling, executing, and debugging programs with both CHERI enabled
+and default ISAs, as well as to build basic understanding of CHERI
+C/C++ properties. We highlight some key edge cases in CHERI, including
+the effects of bounds imprecision, subobject bounds, weaker temporal
+safety, and C type confusion.
 
 These exercises take for granted a strong existing understanding of:
 - The C/C++ languages
 - UNIX program compilation, execution, and debugging
-- RISC ISAs and binary structures/reverse engineering (e.g., on MIPS or ARMv8)
+- ARMv8 and/or RISC ISAs as well as binary structures/reverse engineering
 
 ## Focused adversarial missions
 **Focused adversarial missions** are intended to take
-1-3 days, and ask you to exploit, first on RISC-V, and
-then on CHERI-RISC-V, documented vulnerabilities in simple "potted"
-C/C++-language programs provided by the CHERI-RISC-V team. These missions
-engage you more specifically in RISC-V exploitation, and CHERI's
+1-3 days, and ask you to exploit, first on an unmodified ISA, and
+then on a CHERI enabled ISA, documented vulnerabilities in simple "potted"
+C/C++-language programs provided by the CHERI team. These missions
+engage you more specifically in exploitation, and CHERI's
 security objectives and mechanisms.
 
 **These take for granted good existing experience with
 memory-safety-related attack techniques, such as buffer overflows,
 integer-pointer type confusion, Return-Oriented Programming (ROP), and
 Jump-Oriented Programming (JOP).**
 
-Successful exploitation of RISC-V variants depends only upon
-widely published understanding and techniques (e.g., buffer overflows
-combined with ROP). For those familiar with conventional low-level
-attack techniques, this will also act as a warm-up exercise on the
-baseline RISC-V architecture and expand experience with RISC-V reverse
+Successful exploitation of non CHERI enabled variants depends only
+upon widely published understanding and techniques (e.g., buffer
+overflows combined with ROP). For those familiar with conventional
+low-level attack techniques, this will also act as a warm-up exercise
+on the baseline architecture and expand experience with reverse
 engineering and exploitation.
 
-The CHERI-RISC-V team has confirmed exploitability for the RISC-V binary
-in advance.  We strongly recommend exploiting the RISC-V version of the code
-first, as a starting point for understanding potential CHERI-RISC-V
-exploitability.
+The CHERI team has confirmed exploitability for the non CHERI enabled
```

Comment:
baseline?

Reply:
Yeah, it's awkward but I don't think baseline works either.  We can revisit in a bit.